src/App.jsx

```diff
@@ -1,107 +1,28 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUsersFromServer(userId) {
```

Comment:
```suggestion
function getUserFromServer(userId) {
```

This function return only one user